connector/spanmetricsconnector/config.go

```diff
@@ -84,9 +85,16 @@ type HistogramConfig struct {
 	Explicit    *ExplicitHistogramConfig    `mapstructure:"explicit"`
 }
 
+type ExemplarDynamicExportConfig struct {
+	Enabled bool `mapstructure:"enabled"`
+	// Attribute name to be used to check if exemplar should be stored.
+	AttributeKeyName string `mapstructure:"attribute_key_name"`
+}
+
 type ExemplarsConfig struct {
-	Enabled         bool `mapstructure:"enabled"`
-	MaxPerDataPoint *int `mapstructure:"max_per_data_point"`
+	Enabled         bool                         `mapstructure:"enabled"`
+	MaxPerDataPoint *int                         `mapstructure:"max_per_data_point"`
+	DynamicConfig   *ExemplarDynamicExportConfig `mapstructure:"dynamic"`
```

Comment:
Having this as a pointer doesn't provide much value due to the fact that `ExamplarDynamicExportConfig` has the field `Enabled` which will be false as the zero value.